src/meta/assessment/section.ts

```diff
@@ -4,6 +4,11 @@ import { Label } from './label'
 
 export type SectionName = string
 
+export enum SectionNames {
```

Comment:
Enum is called `SectionNames` because of the conflicting `SectionName`

Should we change this or leave?